components/SSI/DIDxWallet/Wallet/NFTUsername/ManageNFT/TransferNFTUsername/index.tsx

```diff
@@ -124,16 +124,89 @@ function Component() {
                         : input
                 const tyron_ = await tyron.Donation.default.tyron(donation!)
 
-                const tx_params =
```

Comment:
why are you going backwards @DevPH-Wistkey @Sherly09

Reply:
> why are you going backwards @DevPH-Wistkey @Sherly09

@tralkan would you mind explaining more about what is the issue for this?

Reply:
Hi @tralkan , I commenting the tx params from tyron.js so that you can compare it. Because when I sending doc version 5.5 to tyron.js, seems like it didn't read properly. But let me put back again the tx to tyron.js